.github/workflows/test.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: build cpp
         run: |
-          cd trino-cpp
+          cd Gluten-Trino-Cpp
```

Comment:
Hmm, will cpp be more simple as the repo name already specifics Gluten-Trino?

Reply:
maybe `cpp` as a dir name a little confusing?

Reply:
How do we plan to name ```GlutenTrinoPlugin``` folder? Two options on top of mind: 1. top level folders as java, cpp like [Arrow](https://github.com/apache/arrow); 2. Gluten-like, which root folder has a maven pom. I am more preferring to use option 1. 

Thoughts?

Reply:
renamed as option 1